docs/source/noaa_csp.rst

```diff
@@ -24,177 +23,398 @@ resolution.
      - **Instance Type**
      - **Partition**
      - **File System**
-   * - Amazon Web Services Parallel Works
-     - C48
-     - ``ATM``
-     - ``c5.9xlarge (36 vCPUs, 72 GB Memory, amd64)``
+   * - Amazon Web Services ParallelWorks
+     - C48, C96, C192, C384
+     - ``ATM``, ``GEFS``
+     - ``c5.18xlarge (72 vCPUs, 144 GB Memory, amd64)``
      - ``compute``
-     - ``/lustre``
+     - ``/lustre``, ``/bucket``
+   * - Azure ParallelWorks
+     - C48, C96, C192, C384
+     - ``ATM``, ``GEFS``
+     - ``Standard_F48s_v2 (48 vCPUs, 96 GB Memory, amd64)``
+     - ``compute``
+     - ``/lustre``, ``/bucket``
+   * - GCP ParallelWorks
+     - C48, C96, C192, C384
+     - ``ATM``, ``GEFS``
+     - ``c3d-standard-60-lssd (60 vCPUs, 240 GB Memory, amd64)``
+     - ``compute``
+     - ``/lustre``, ``/bucket``
 
-Instructions regarding configuring the respective CSP instance and
-cluster follows.
+Instructions regarding configuring the respective CSP instances and
+clusters follow.
 
 *********************
 Login to the NOAA CSP
 *********************
 
-Log in to the `NOAA CSP <http://noaa.parallel.works/login>`_ and into
+Log in to the `NOAA cloud <http://noaa.parallel.works/login>`_, and into
 the resources configuration. The user should arrive at the following
-screen.
+screen as in :numref:`Figure %s <pw-home>`.
+Click the :blue-text:`blue` box indicated by the red arrow to login.
```

Comment:
```suggestion
Click the blue box indicated by the red arrow to login.
```
`:blue-text:` and `:red-text:` won't work as intended unless you add styling to `docs/source/_static/custom.css`

Reply:
It was defined in : docs/source/custom.css
Let us remove it for now.
Thanks.